types/test/model.ts

```diff
@@ -67,3 +67,19 @@ UserModel.findCreateFind({
   }
 })
 
+class ModelCompositePrimaryKey extends Model {}
+
+ModelCompositePrimaryKey.init({
+  uid: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+  primaryKeyOne: { type: DataTypes.STRING, primaryKey: true, allowNull: false },
+  title: DataTypes.STRING,
+  description: DataTypes.TEXT,
+}, { sequelize, modelName: 'ModelCompositePrimaryKey' });
+
+ModelCompositePrimaryKey
+  .create({ primaryKeyOne: "Type 1", title: 'Test Record', description: 'Record created to test composite primary key' })
+  .then(modelCompositePrimaryKey => {
+    console.log(modelCompositePrimaryKey.get('title'));
+    console.log(modelCompositePrimaryKey.get('description'));
+    console.log(ModelCompositePrimaryKey.primaryKeyAttributes);
+  })
```

Comment:
As I said above, please simplify all this to

```ts
class TestModel extends Model {};
TestModel.primaryKeyAttributes;
```

Reply:
Hi @papb, I understand my test was too complicated. Thank you for the thorough explanation. I have made the changes. Please let me know if the changes are ok.